modules/ROOT/partials/integrations/svelte-quick-start.adoc

```diff
@@ -140,7 +139,7 @@ let conf = {
 <main>
   <h1>Hello Tiny</h1>
   <Editor
-    licenseKey='your-license-key'
+    licenseKey='gpl', // gpl for open source, T8LK:... for commercial
```

Comment:
```suggestion
    licenseKey='gpl'
```

Please make sure the syntax is correct

Reply:
Not sure what you mean here, luke and I both agree that `// gpl for open source, T8LK:... for commercial` is a good idea so integrators are aware of the requirements for commercial.

Reply:
@kemister85 are you really able to use a comma and comment like this in the middle of a block of HTML?